From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Jeremy Apthorp <nornagon@nornagon.net>
Date: Mon, 10 Feb 2020 10:37:48 -0800
Subject: use electron resources in pdf_util

Without this, the ID for IDR_PDF_MANIFEST will be wrong on linux
and cause a DCHECK(), since the resource will be loaded as an empty
string.

diff --git a/chrome/browser/pdf/pdf_extension_util.cc b/chrome/browser/pdf/pdf_extension_util.cc
index 48ec3753709a971fafe4e194dce54402957ab6c4..132593076a5a7770cfb2351e8604364aed88f00c 100644
--- a/chrome/browser/pdf/pdf_extension_util.cc
+++ b/chrome/browser/pdf/pdf_extension_util.cc
@@ -8,7 +8,7 @@
 #include "base/values.h"
 #include "chrome/browser/browser_process.h"
 #include "chrome/common/chrome_content_client.h"
-#include "chrome/grit/browser_resources.h"
+#include "electron/grit/electron_resources.h"
 #include "components/strings/grit/components_strings.h"
 #include "components/zoom/page_zoom_constants.h"
 #include "pdf/pdf_features.h"
